zenpacklib.py

```diff
@@ -6315,6 +6315,13 @@ def create_zenpack_srcdir(zenpack_name):
     import shutil
     import errno
 
+    # validate ZP name - it must begins with "ZenPacks.*"
+    if not zenpack_name.startswith('ZenPacks.'):
+        sys.exit(
+            "`{}` is a wrong name for ZenPack - zenpack name must starts "
+            "with `ZenPacks`. Please, check ZP name.".format(zenpack_name)
+        )
```

Comment:
I agree that it's a good idea to perform this check and also to provide direct feedback.  The main issue with this PR, though, is that the monolithic zenpacklib.py file is now vestigial and is not the appropriate place for the change.  

I took your suggestion and created a similar change here:
https://github.com/zenoss/ZenPacks.zenoss.ZenPackLib/pull/420
and also added a check for the length of the ZenPack namespace (since that might go wrong as well).